README.md

```diff
@@ -38,6 +43,67 @@ If you need support, please either raise an issue on Github or ask on [Slack](ht
 
 Substra warmly welcomes any contribution. Feel free to fork the repo and create a pull request.
 
+## How to test
+
+Install `substrafl` in editable mode with developper dependencies.
+In addition, install `substra` and `substra-tools` in editable mode.
+Clone this repo, then at the top level run in your virtual env:
+
+```sh
+pip install -e ".[dev]"
+``
+
+Clone [`substra`](https://github.com/Substra/substra) locally, go to the top level directory of `substra` and run (still in your virtual env):
```

Comment:
```suggestion
Clone [`substra`](https://github.com/Substra/substra) locally, go to the top level directory of `substra` and run:
```
I would add a line in the readme about virtual env, but not precise it every time